autoload/clap/provider/git_files.vim

```diff
@@ -6,14 +6,32 @@ set cpoptions&vim
 
 let s:git_files = {}
 
-if executable('git')
-  let s:git_files.source = 'git ls-files '.(has('win32') ? '' : ' | uniq')
-else
-  let s:git_files.source = ['git executable not found']
-endif
+function! s:git_files.source() abort
```

Comment:
Please move the open action support for git_files in another PR.

Reply:
Whoops, sorry. Done!